lib/ppx_mysql_runtime.ml

```diff
@@ -1,14 +1,10 @@
 (********************************************************************************)
 (** {1 Public functions and values}                                             *)
+
 (********************************************************************************)
 
 let identity x = x
+let map_option f = function Some x -> Some (f x) | None -> None
+let get_option = function Some x -> x | None -> assert false
```

Comment:
I don't like this compared to what was before.

Reply:
I agree. I made it break all pattern matches. Maybe it is better now?